operator/v1/types_ingress.go

```diff
@@ -308,6 +316,33 @@ type EndpointPublishingStrategy struct {
 	NodePort *NodePortStrategy `json:"nodePort,omitempty"`
 }
 
+// RouteAdmissionPolicy is an admission policy for allowing new route claims.
+type RouteAdmissionPolicy struct {
+	// namespaceOwnership describes how host name claims across namespaces should
+	// be handled.
+	//
+	// Value must be one of:
+	//
+	// - Strict: do not allow routes to claim the same host name across namespaces.
```

Comment:
It might be helpful to spell out in at least one instance what the phrase "across namespaces" means; for example, "Strict: Do not allow routes in different namespaces to claim the same host."

Reply:
Good idea, changed